src/components/plotly/shared/styles/legend.js

```diff
@@ -37,18 +36,22 @@ export default {
     alignItems: 'center',
   }),
 
-  LegendColorBox: styled('div')(({ color, rightAligned }) => ({
-    order: + !rightAligned,
-    background: color,
-    borderRadius: '0.2rem',
-    width: '1rem',
-    height: '1rem',
-    margin: '0 0.5rem',
-  })),
+  LegendColorBox: styled('div')(({ rightAligned, color }) => {
```

Comment:
@parkdoeui Not sure why we changed syntax here. The above should have worked perfectly and it is simplified. I prefer you not change it.

Reply:
I don't see any syntax change other than replacing `DivBase` to `div`. Can you please be more specific on what part of syntax are you referring to?

Reply:
That was from the test whether the destructured objects cause the issue. Let me revert them